lib/Rest/VstResponse.h

```diff
@@ -34,6 +34,9 @@ class VstResponse : public GeneralResponse {
  public:
   VstResponse(ResponseCode code, uint64_t mid);
 
+  virtual bool isResponseEmpty() const override {
```

Comment:
```suggestion
 bool isResponseEmpty() const override {
```
only parent class needs to be virtual